### PR TITLE
add Alex to GH mgmt stewards

### PR DESCRIPTION
**Pull request by galargh**
### What else do we need to know?

**DRI:** myself

### Reviewer's Checklist

- [ ] It is clear where the request is coming from (if unsure, ask)
- [ ] All the automated checks passed
- [ ] The YAML changes reflect the summary of the request
- [ ] The Terraform plan posted as a comment reflects the summary of the request

### CODE REVIEW

File: github/libp2p.yml [hunk 1]
```diff
@@ -4166,6 +4166,9 @@ repositories:
     default_branch: main
     delete_branch_on_merge: true
     description: Use a Reframe service to discover content providers
+    files:
```
Comment (achingbrain) [inline github/libp2p.yml:4169]: Unrelated change?

**Review by achingbrain — APPROVED** [review]

File: github/libp2p.yml (hunk 1 above)
Comment (galargh) [reply github/libp2p.yml:4169]: It's an automatic fix, because I didn't merge https://github.com/libp2p/github-mgmt/pull/169 before. It'll get rebased in a sec. I'll merge afterwards.

**Review by p-shahi — APPROVED** [review]
